src/OpenTelemetry.Instrumentation.AspNet/AspNetInstrumentationOptions.cs

```diff
@@ -41,5 +42,10 @@ public class AspNetInstrumentationOptions
         /// If Filter returns false or throw exception, the request is filtered out.
         /// </summary>
         public Func<HttpContext, bool> Filter { get; set; }
+
+        /// <summary>
+        /// Gets or sets an action to enrich an Activity.
```

Comment:
can you add a basic comment explaining what the parameters to the Action?
Activity - the activity being enriched.
string - name of the event, which gives hint about the next parameter.
object  - the raw object from which additional information can be extracted to enrich the activity. The type of this object depends on the event, which is given by the above parameter.

Reply:
just updated.